src/librustdoc/html/static/main.js

```diff
@@ -363,6 +363,7 @@ function defocusSearchBar() {
     function handleEscape(ev) {
         var help = getHelpElement();
         var search = getSearchElement();
+        hideModal();
```

Comment:
Why did you add the `hideModal` calls BTW?

Reply:
Didn't intend to. Will fix.